content/experts/map.toml

```diff
@@ -259,3 +259,12 @@ crates = [
 experts = ["michaelwoerister", "wesleywiser"]
 familiar = []
 
+[[areas]]
+name = "Rustdoc"
+directories = [
+  "src/librustdoc",
+  "src/tools/rustdoc",
+]
+crates = []
+experts = ["GuillaumeGomez"]
```

Comment:
Please add @ollie27 too? Also, maybe @Manishearth and @kinnison too? Depends on how they feel about it.

Reply:
I'm okay with being added

Reply:
You are listed in "familiar" currently but I thought that you were more an expert. So it's all up to you.

Reply:
I'm definitely only familiar, not expert :D

Reply:
I added @ollie27 to `experts` and moved @Manishearth there from `familiar`.